src/HomeWorkApp.java

```diff
@@ -0,0 +1,81 @@
+public class HomeWorkApp {
+
+    public static void main(String[] args) {
+//         printThreeWords();
+//         checkSumSign(-10,-5);
+//         printColor(-1);
+//         compareNumbers(5,6);
+//         System.out.println(trueOrFalse(5,5));
+//         negativeOrPositive(-5);
+//         System.out.println(positiveFalseOrNegativeTrue(-5));
+//          printStringNTimes("Hello", 5);
+            System.out.println(isLeapYear(100));
+
+    }
+
+    static void printThreeWords() {// метод который выводит 3 слова в табличном виде
+        System.out.println(
+                """
+                        Orange
+                        Banana
+                        Apple
+                        """
+        );
+    }
+
+    static void checkSumSign(int a, int b) { // метод который считает положительна или отрицательна сумма
+        int c = a + b;
+        if (c >= 0) {
+            System.out.println("Сумма положительна!"); // если сумма чисел больше или равна 0 вывести Сумма положительна
+        } else {
+            System.out.println("Сумма отрицательна"); // в противном случае вывести Сумма отрицательна
+        }
+    }
+
+    static void printColor(int value) { // метод который выводит цвет, в зависимости от переданного значения
+        if (value <= 0) {
+            System.out.println("Красный"); // если value меньше или ровно 0 вывести слово Красный
+        } else if (value > 0 && value <= 100) {
```

Comment:
Эта проверка уже выполнена выше. Не стоит усложнять решение
```suggestion
        } else if (value <= 100) {
```